docs/Containers/Octoprint.md

```diff
@@ -478,3 +467,4 @@ $ docker-compose up -d octoprint
 ```
 
 The OctoPrint container is well-behaved and will re-initialise its persistent storage area correctly. OctoPrint will adopt "first run" behaviour and display the Setup Wizard.
+bbedit 
```

Comment:
This a typo?

Reply:
Yes. Brain flatulence. As the PR still seems to be open, I assume I can fix-and-push. I'll do that now.